src/components/static-map.js

```diff
@@ -232,24 +189,35 @@ export default class StaticMap extends PureComponent {
   }
 
   // External apps can access map this way
-  _getMap() {
+  getMap() {
```

Comment:
Hm actually. I see that we added an `onLoad` prop to here. Does it make sense to have both a `getMap()` method and an `onLoad` method? Can we just return the map instance for `onLoad` as well? I feel that it'll be more react-like that way, rather than exposing an external method.